canvasapi/account.py

```diff
@@ -1783,6 +1800,16 @@ def update_account_auth_settings(self, **kwargs):
 
         return SSOSettings(self._requester, response.json())
 
+    def update_account_calendars_visibility(self, **kwargs):
+        """
+        Update many account calendars visibility at once.
+
+        :calls: `PUT /api/v1/accounts/:account_id/account_calendars \
+        <https://canvas.instructure.com/doc/api/account_calendars.html#method.account_calendars_api.bulk_update>`_
+
+        :rtype: :class:`canvasapi.account.accountCalendar`
+        """
+
```

Comment:
Please fill out this function or remove it. At the very least, make the body of the function a `pass` statement so the file doesn't error.